ElectronClient/ElectronAppWrapper.js

```diff
@@ -43,9 +43,12 @@ class ElectronAppWrapper {
 
 		const windowStateKeeper = require('electron-window-state');
 
+		const currentWorkArea = screen.getPrimaryDisplay().workArea;
+		const [defaultWidth, defaultHeight] = [0.8*currentWorkArea.width, 0.8*currentWorkArea.height];
```

Comment:
Please round the values. Also I don't see a need to create these separate values, so if you could set the properties on stateOptions directly that would be best.

Reply:
@laurent22 can you take a look at the changes?